### ISSUE TITLE
some external links are broken

### ISSUE THREAD

**Issue by ydcjeff** [pytorch/ignite#1704]
## 📚 Documentation

Sphinx has a builder for checking external links. You can run with `make linkcheck` in the docs directory. And, there are some broken links.
```
about.rst:22: [broken] governance.html: 
contrib/handlers.rst:93: [broken] https://docs.neptune.ai/python-api/tutorials/get-started.html#copy-api-token: Anchor 'copy-api-token' not found
contrib/handlers.rst:72: [broken] https://docs.polyaxon.com/references/polyaxon-tracking-api/experiments/: 404 Client Error: Not Found for url: https://polyaxon.com/docs/references/polyaxon-tracking-api/experiments/
contrib/handlers.rst:72: [broken] https://docs.polyaxon.com/references/polyaxon-tracking-api/experiments/: 404 Client Error: Not Found for url: https://polyaxon.com/docs/references/polyaxon-tracking-api/experiments/
contrib/handlers.rst:76: [broken] https://github.com/facebookresearch/visdom#visdom-arguments-python-only: Anchor 'visdom-arguments-python-only' not found
contrib/handlers.rst:139: [broken] https://docs.polyaxon.com/references/polyaxon-tracking-api/experiments/: 404 Client Error: Not Found for url: https://polyaxon.com/docs/references/polyaxon-tracking-api/experiments/
distributed.rst:34: [broken] https://horovod.readthedocs.io/en/latest/api.html#horovod.torch.init: Anchor 'horovod.torch.init' not found
engine.rst:113: [broken] https://github.com/pytorch/ignite/tree/master/examples/contrib/cifar10#check-resume-training: Anchor 'check-resume-training' not found
engine.rst:242: [broken] https://github.com/pytorch/ignite/tree/master/examples/contrib/cifar10#check-resume-training: Anchor 'check-resume-training' not found
engine.rst:112: [broken] https://github.com/pytorch/ignite/tree/master/examples/mnist#training-save--resume: Anchor 'training-save--resume' not found
engine.rst:241: [broken] https://github.com/pytorch/ignite/tree/master/examples/mnist#training-save--resume: Anchor 'training-save--resume' not found
faq.rst:85: [broken] _modules/ignite/contrib/engines/tbptt.html#create_supervised_tbptt_trainer: 
```
To have an updated link to check, we could do
- add a github action job in `docs.yml` checking `make linkcheck` running on every PR
- update the links with new links

**Comment by vfdev-5** [comment]
That's will be a great addition, thanks @ydcjeff ! Do you know if there is an option to check internal links as well ?

**Comment by ydcjeff** [comment]
Yes, there is (it's called nitpicky mode).
You can check by adding `SPHINXOPTS="-n"` to `make html`.

But, since `autodoc` finds type hints to the full module path (i.e, `ignite.handlers.Checkpoint` becomes `ignite.handlers.checkpoint.Checkpoint`), checking for internal links are failing currently. May be a patch can do that, I can't figure it out currently.

**Comment by mouradmourafiq** [comment]
@ydcjeff the polyaxon related URLs will be fixed in #1119
